src/Exceptions/IndodanaRequestException.php

```diff
@@ -2,42 +2,43 @@
 
 namespace Indodana\Exceptions;
 
-class IndodanaRequestException extends \Exception {
+class IndodanaRequestException extends \Exception
+{
   private $kind;
   private $errorMessage;
 
-  public function __construct(array $response = []) {
-    $error = $response['error'];
-
-    if (empty($error)) {
+  public function __construct(array $response = [])
+  {
+    if (!isset($response['error'])) {
       throw new IndodanaSdkException('Received empty error from Indodana');
```

Comment:
Can you add other details here? such as status code or the response text so at least there's a clue why it's empty error e.g. because request timeout

Reply:
Alright!